Lib/sqlite3/__main__.py

```diff
@@ -105,17 +112,21 @@ def main(*args):
         Each command will be run using execute() on the cursor.
         Type ".help" for more information; type ".quit" or {eofkey} to quit.
     """).strip()
-    sys.ps1 = "sqlite> "
-    sys.ps2 = "    ... "
+
+    theme = get_theme()
+    s = theme.syntax
+
+    sys.ps1 = f"{s.prompt}sqlite> {s.reset}"
+    sys.ps2 = f"{s.prompt}    ... {s.reset}"
 
     con = sqlite3.connect(args.filename, isolation_level=None)
     try:
         if args.sql:
             # SQL statement provided on the command-line; execute it directly.
-            execute(con, args.sql, suppress_errors=False)
+            execute(con, args.sql, suppress_errors=False, theme=theme)
         else:
             # No SQL provided; start the REPL.
-            console = SqliteInteractiveConsole(con)
+            console = SqliteInteractiveConsole(con, use_color=True)
```

Comment:
While this is `True`, the environment might still lead to no color being emitted. I think this is okay for this use case, PyREPL behaves the same way.